_extras/guide.md

```diff
@@ -27,13 +27,15 @@ for a look at how using checklists can save lives (and make many other things be
 
 > ## Setting Up [Trainer]
 >
+> 1.  Create an Etherpad (http://pad.software-carpentry.org/<nameofevent>
```

Comment:
Yes, there have been. I think the important point to make would be that the link to the google doc should still be pasted in the "etherpad" section of the workshop template. 

Have you ever used google docs? Want to take a stab at adding those instructions?